### PR TITLE
move svgs for catalog

### PR DESCRIPTION
**Pull request by SirSpidey**
### Description

Delete the `svgs` directory and files in favor of the same svgs in the `/reference-architectures` directory. I've left the drawio file where it is

The svgs are used in the reference architectures that are published to Cloud Docs, and Docs needs them in this location.

:exclamation: **Important:** This move means that the reference in the Catalog tile needs to change.

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

**Review by ocofaigh — APPROVED** [review]

**Comment by ocofaigh** [comment]
@SirSpidey Just noticed you missed the VSI standard ibm_catalog.json. I have updated it in https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/pull/297

File: ibm_catalog.json [hunk 1]
```diff
@@ -275,7 +275,7 @@
                             {
                                 "diagram": {
                                     "caption": "Secure infrastructure on VPC for regulated industries",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/main/patterns/diagrams/svgs/vsi/vsi-vsi.drawio.svg",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/terraform-ibm-landing-zone/patterns/diagrams/svgs/vsi/vsi-vsi.drawio.svg",
```
Comment (ocofaigh) [inline ibm_catalog.json:278]: This is not correct - Ill fix in https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/pull/297

File: ibm_catalog.json [hunk 2]
```diff
@@ -462,7 +462,7 @@
                             {
                                 "diagram": {
                                     "caption": "Secure infrastructure on VPC with ROKS",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/main/patterns/diagrams/svgs/roks/roks.drawio.svg",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/terraform-ibm-landing-zone/patterns/diagrams/svgs/roks/roks.drawio.svg",
```
Comment (ocofaigh) [inline ibm_catalog.json:465]: This is not correct - Ill fix in https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/pull/297

Comment (SirSpidey) [reply ibm_catalog.json:465]: Thanks for catching both, @ocofaigh. Sorry, moving too fast.

**Pull request merged**